.github/ISSUE_TEMPLATE/release.md

```diff
@@ -13,33 +13,34 @@ labels: ["release"]
 
 ### At the beginning of the cycle
 
-- [ ] Create a new `release-notes-v0.4x.0` branch and add a new release notes file using the available [template](/grafana/k6/tree/master/release%20notes/template.md) to the [repository's `release notes` folder](/grafana/k6/tree/master/release%20notes).
+- [ ] Get from `@k6-browser` team the Release coordinator on their side and co-assign the issue.
+- [ ] Create a new `release-notes-v0.4x.0` long-lived branch and add a new release notes file using the available [template](/grafana/k6/tree/master/release%20notes/template.md) to the [repository's `release notes` folder](/grafana/k6/tree/master/release%20notes).
 - [ ] Go through the potential [dependencies updates](Dependencies.md) and create a dedicated PR if any of them is relevant to this release.
+- [ ] Create a new `release-v0.4x.0` long-lived branch on the [k6-docs](https://github.com/grafana/k6-docs) repository. Try to keep the branch synced with `main` during the cycle for not getting an outdated branch and get a lot of conflicts.
```

Comment:
```suggestion
- [ ] Create a new `release-v0.4x.0` long-lived branch on the [k6-docs](https://github.com/grafana/k6-docs) repository. Try to keep it synchronized with `main` throughout the development cycle to prevent it from becoming outdated and to avoid numerous conflicts.
```